pkg/v1/empty/image.go

```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/types"
 )
 
+// docker scratch image name
+const Scratch = "scratch"
```

Comment:
I'd like to avoid adding an exported field to this package until we get more signal that it's useful. In the meantime just matching against the string in rebase.go sounds fine to me.